command/quickdeploy.go

```diff
@@ -23,6 +23,7 @@ Examples:
 
   force quickdeploy -v 0Af0b000000ZvXH
 `,
+	MaxExpectedArgs: -1,
```

Comment:
Shouldn't it be 1?

Reply:
There is no consistency between force commands, so I think we should leave the existing behaviour since it already reports an error if the number or arguments don't match.